samples/CommunityToolkit.Maui.Sample/AppShell.xaml.cs

```diff
@@ -93,9 +96,17 @@ public static string GetPageRoute(Type viewModelType)
 
 	static void RegisterRouting()
 	{
-		foreach(var viewModelKeyValuePair in viewModelMappings)
+		foreach (var viewModelKeyValuePair in viewModelMappings)
 		{
 			Routing.RegisterRoute(GetPageRoute(viewModelKeyValuePair.Value.GalleryPageType, viewModelKeyValuePair.Value.ContentPageType), viewModelKeyValuePair.Value.ContentPageType);
```

Comment:
You can see TGalleryViewModel is not used here.

Reply:
Right! Here's more background + detail on my rationale behind the implementation: https://github.com/CommunityToolkit/Maui/pull/317#discussion_r820143745